src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -5086,6 +5092,8 @@ CalcHeatBalanceInsideSurf( Optional_int_const ZoneToResimulate ) // if passed in
 			// check for saturation conditions of air
 			Real64 const HConvIn_surf( HConvInFD( SurfNum ) = HConvIn( SurfNum ) );
 			RhoVaporAirIn( SurfNum ) = min( PsyRhovFnTdbWPb_fast( MAT_zone, ZoneAirHumRat_zone, OutBaroPress ), PsyRhovFnTdbRh( MAT_zone, 1.0, HBSurfManInsideSurf ) );
+			// TODO: Check with Jason about why RhoVaporAirIn( SurfNum ) wasn't added in his version.
+			//       The original was commented out, which makes me think it was a debugging effort.
```

Comment:
@jdwoods21 Your version had the `RhoVaporAirIn( SurfNum )` removed, should this be deleted here too?